RELEASE_MANAGEMENT.md

```diff
@@ -0,0 +1,107 @@
+# Release Management
+
+Major and minor releases for Gateway API are managed by a "Release Manager".
+The responsibilities of the release manager include:
+
+* Creating and managing a GitHub Milestone for the release.
+* Creating and managing a GitHub Project board for the release.
+* Creating and managing a GitHub Discussion Boards announcement for the release.
+* Working through the [Release Cycle](#release-phases) for the release.
+
+This management process ultimately results in the manager of the release
+shipping the release as per the [Release Process]. We will go into more detail
+about this in the sections that follow.
+
+> **Note**: Patch releases need someone assigned to them as well, however they
+> just happen organically, and as soon as possible. We currently we don't bother
+> sending patch releases through this entire process.
+
+[Release Cycle]:https://gateway-api.sigs.k8s.io/contributing/release-cycle/
+[Release Process]:/RELEASE.md
+
+## Assigning a Release Manager
+
+The [maintainers] will find and assign a release managers for upcoming
+releases. A release manager can be a community member on the contributor ladder
+who volunteers (at the discretion of the maintainers), or otherwise the
+responsibility falls back to the maintainers.
+
+[maintainers]:/OWNERS_ALIASES
+
+## GitHub Milestone
+
+Once a manager is assigned for the release, they must create a [GitHub
+Milestone] for the release, where the name of the milestone is the eventual tag
+that will be cut (e.g., `v1.0.0`).
+
+The release manager must mark themselves as the manager for the release at the
+top of the milestone's description, and add any description to the milestone
+that communicates to the community any major themes in the release, and
+important notes:
+
+```console
+release-manager: @<name>
+
+This milestone introduces the following new experimental features:
+
+* <experimental-feature-1>
+* <experimental-feature-2>
+* <experimental-feature-3>
+
+This milestone graduates the following features:
+
+* <GA-feature-1>
+* <GA-feature-2>
+
+feature-freeze: <mm-dd-yy>
+```
+
+> **Note**: It's normal for this description to change over time, as things are
+> never completely certain at the onset.
+
+The release manager must assign a due date for the milestone, to communicate to
+the community the intended date to ship the release. They must also indicate an
+expected [feature freeze](#freeze-phase) date in the description.
```

Comment:
Which freeze? We currently have 3 for the end of each significant phase:

1) Scoping
2) GEP
3) API

One of the things we discussed in recent GW API meetings is that we want to set dates for all significant milestones in a release up front, similar to [upstream](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.33#timeline). I'm assuming that would also be defined by the release manager?

Reply:
Let's just remove the confusing language.

```suggestion
```